test_themes/tests/test_new_page_templates.py

```diff
@@ -110,7 +110,7 @@
     's_carousel_indicators_dots', 's_quotes_carousel', 's_rating_no_title',
     's_carousel_indicators_hidden', 's_blog_post_big_picture',
     's_newsletter_list', 's_event_upcoming_snippet', 's_event_event_picture',
-    's_newsletter_subscribe_form',
+    's_newsletter_subscribe_form', 's_attributes_img', 's_attributes_col',
```

Comment:
Adding items to this list means you are doing something wrong: indeed, I missed it in my community review but they should be called `s_attributes_horizontal_img` and `s_attributes_horizontal_col`. Or don't add the word "horizontal" in your snippet name in the first place, as you wish :shrug:

Reply:
As we will also introduce another snippet called `s_attributes_vertical`, I'd prefer to keep the word "horizontal" 🙂  I've made the changes, could you take a look at it?
Thanks in advance!